Assets/Scripts/New/MainMenu.cs

```diff
@@ -82,6 +82,7 @@ private void Awake()
         private void OnClickStartButton()
         {
             homeScreen.SetActive(false);
+            GameService.Instance.Initialize();
             gameplayPanel.SetActive(true);
```

Comment:
Nope, A method called Initialize() in GameService is being called from some other Script, this is not a good thing, Game Service is the ignition for all other Services.